### ISSUE TITLE
Telegram fails to start after changing linux system username

### ISSUE THREAD

**Issue by atr0p1ne** [telegramdesktop/tdesktop#26480]
### Steps to reproduce

1. usermod -l newname -d /home/newname -m oldname
2. restart pc

### Expected behaviour

I think Telegram should resolve this issue by itself..

### Actual behaviour

Error message pop up.. 

[2023.07.05 16:52:22] Executable path before check: /usr/bin/telegram-desktop
[2023.07.05 16:52:22] Launched version: 4008004, install beta: [FALSE], alpha: 0, debug mode: [FALSE]
[2023.07.05 16:52:22] Executable dir: /usr/bin/, name: telegram-desktop
[2023.07.05 16:52:22] Initial working dir: /home/username/.local/share/TelegramDesktop/
[2023.07.05 16:52:22] Working dir: /home/username/.local/share/TelegramDesktop/
[2023.07.05 16:52:22] Command line: /usr/bin/telegram-desktop -workdir /home/username/.local/share/TelegramDesktop/ -autostart
[2023.07.05 16:52:22] FATAL: Could not open '/home/username/.local/share/TelegramDesktop/log_startXX.txt' for writing log!
[2023.07.05 16:52:22] Launcher filename: org.telegram.desktop.desktop
[2023.07.05 16:52:22] Connecting local socket to fe168d2d3d1ccf39ea7891c68add5428-{87A94AB0-E370-4cde-98D3-ACC110C5967D}...
[2023.07.05 16:52:22] Socket connect error 0, starting server and app...
[2023.07.05 16:52:22] Font: from ':/gui/fonts/DAOpenSansRegular.ttf' loaded 'DAOpenSansRegular'
[2023.07.05 16:52:22] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2023.07.05 16:52:22] Font: from ':/gui/fonts/DAOpenSansRegularItalic.ttf' loaded 'DAOpenSansRegularItalic'
[2023.07.05 16:52:22] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2023.07.05 16:52:22] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2023.07.05 16:52:22] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2023.07.05 16:52:22] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2023.07.05 16:52:22] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2023.07.05 16:52:22] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2023.07.05 16:52:22] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2023.07.05 16:52:22] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2023.07.05 16:52:22] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'

### Operating system

Arch Linux

### Version of Telegram Desktop

4.8.4-2

### Installation source

Other (unofficial) source

**Comment by ilya-fedin** [comment]
The meaning of -workdir has changed over years and the modern code doesn't save it unless it was used to start the instance, so it seems to be a shortcut generated by older code. The only way to get rid of it is probably to update the shortcut on every update but I haven't got a permission to do this in cross-platform code and I don't think Linux is any specifc here to do this in Linux specific code.

In other words, I disagree with
> I think Telegram should resolve this issue by itself..

and think the issue should be closed with wontfix as manual intervention is the solution here.
